src/tachimawari/imu/node/imu_node.cpp

```diff
@@ -46,10 +47,13 @@ void ImuNode::update_imu()
   keisan::Vector<3> gyro = imu_provider->get_gyro();
   keisan::Vector<3> accelero = imu_provider->get_accelero();
 
-  unit_msg.gyro = {static_cast<float>(gyro[0]), static_cast<float>(gyro[1]),
-    static_cast<float>(gyro[2])};
-  unit_msg.accelero = {static_cast<float>(accelero[0]), static_cast<float>(accelero[1]),
-    static_cast<float>(accelero[2])};
+  unit_msg.gyro.roll = gyro[0];
+  unit_msg.gyro.pitch = gyro[1];
+  unit_msg.gyro.yaw = gyro[2];
+
+  unit_msg.accelero.roll = accelero[0];
+  unit_msg.accelero.pitch = accelero[1];
+  unit_msg.accelero.yaw = accelero[2];
```

Comment:
1. let's make `imu_provider->get_gyro()` to return `keisan::Euler` and `imu_provider->get_accelero()` to return `keisan::Point3`.
2. `unit_msg.accelero` should have `x, y, z` property instead of `roll, pitch, yaw`.